pages/common/wikiman.md

```diff
@@ -0,0 +1,21 @@
+# wikiman
+
+> Offline search engine for documentation.
+> Supports manual pages, Arch Wiki, Gentoo Wiki, FreeBSD documentation, and tldr-pages.
+> More information: <https://github.com/filiparag/wikiman>.
+
+- Search for a specific topic on all install sources:
+
+`wikiman {{search_term}}`
+
+- Search for a topic on a specific [s]ource:
+
+`wikiman -s {{source}} {{search_term}}`
+
+- Search for a topic on two specific [s]ources:
+
+`wikiman -s {{source1}},{{source2}} {{search_term}}`
+
+- List existing [S]ources:
+
+`wikiman -S`
```

Comment:
```suggestion
`wikiman -S`

- Display [h]elp:

`wikiman -h`
```
Since there's room for another example I think we can include a `-h` example. Either way, this PR looks good to me.